oci/manager.go

```diff
@@ -123,6 +124,19 @@ func (manager *ociManager) Catalog(ctx context.Context, qopts ...packmanager.Que
 	query := packmanager.NewQuery(qopts...)
 	qname := query.Name()
 	qversion := query.Version()
+	platform := &v1.Platform{}
+
+	if query.Architecture() != "" {
+		platform.Architecture = query.Architecture()
+	}
+
+	if query.Platform() != "" {
+		platform.OS = query.Platform()
+	}
+
+	if len(query.Kconfig()) > 0 {
```

Comment:
`s/Kconfig/KConfig/g` to keep it consistent with other instances in the project.